### ISSUE TITLE
Missing button labels in entry flow

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2810]
Received feedback from a researcher who did an analysis of Hubs using a screenreader that there are labels missing from buttons in the entry flow for Hubs. 

From their report: 
 -After clicking “Allow” in the prompt (Figure 11), the microphone is enabled, indicated by a green button (Figure 12). ​None of those buttons is labeled.
- Clicking “Next” open the audio selection interface (Figure 13). ​Buttons are not labeled.
